src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1689,40 +1601,30 @@ FullWidthRepeat:
         End Function
 
         Private Function TokenOfStringCached(spelling As String, Optional kind As SyntaxKind = SyntaxKind.IdentifierToken) As SyntaxKind
-            If spelling.Length = 1 OrElse spelling.Length > 16 Then
-                Return kind
-            End If
-
+            If spelling.Length = 1 OrElse spelling.Length > 16 Then Return kind
             Return _KeywordsObjs.GetOrMakeValue(spelling)
         End Function
 
         Private Function ScanBracketedIdentifier(precedingTrivia As SyntaxList(Of VisualBasicSyntaxNode)) As SyntaxToken
-            Debug.Assert(CanGetChar)  ' [
-            Debug.Assert(PeekChar() = "["c OrElse PeekChar() = FULLWIDTH_LEFT_SQUARE_BRACKET)
+            Debug.Assert(CanGet)  ' [
+            Debug.Assert(Peek() = "["c OrElse Peek() = FULLWIDTH_LEFT_SQUARE_BRACKET)
 
             Dim IdStart As Integer = 1
             Dim Here As Integer = IdStart
 
             Dim InvalidIdentifier As Boolean = False
 
-            If Not CanGetCharAtOffset(Here) Then
-                Return MakeBadToken(precedingTrivia, Here, ERRID.ERR_MissingEndBrack)
-            End If
+            If Not CanGet(Here) Then Return MakeBadToken(precedingTrivia, Here, ERRID.ERR_MissingEndBrack)
 
-            Dim ch = PeekAheadChar(Here)
+            Dim ch = Peek(Here)
 
             ' check if we can start an ident.
-            If Not IsIdentifierStartCharacter(ch) OrElse
-                (IsConnectorPunctuation(ch) AndAlso
-                    Not (CanGetCharAtOffset(Here + 1) AndAlso
-                         IsIdentifierPartCharacter(PeekAheadChar(Here + 1)))) Then
-
-                InvalidIdentifier = True
-            End If
+            If Not IsIdentifierStartCharacter(ch) OrElse (IsConnectorPunctuation(ch) AndAlso Not (CanGet(Here + 1) AndAlso
```

Comment:
Could you please keep the newline after the `OrElse` in the condition.
Now I need to scroll right the diff on GitHub to see the end of the proposed line.